forms-flow-nav/src/constants/constants.js

```diff
@@ -199,7 +199,13 @@ export const DRAFT_ENABLED =
     : false;
 
 // This is to check if the application is running in enterprise mode or not
-export const IS_ENTERPRISE = window._env_?.REACT_APP_IS_ENTERPRISE || false;
+const ENTERPRISE_ENABLED_VARIABLE =
+  (window._env_ && window._env_?.REACT_APP_IS_ENTERPRISE) || false;
+
+export const IS_ENTERPRISE = 
+  ENTERPRISE_ENABLED_VARIABLE === "true" || ENTERPRISE_ENABLED_VARIABLE === true
+    ? true
+    : false;
```

Comment:
Whether there is a need for the ternary (? true : false) because the || condition itself already produces a boolean. Can you please check?